map_frame: map
lidar_frame: PandarQT
camera_frame: camera2_link
rotation:
  [
    0.0,
    0.0,
    0.0,
    1.0,
  ]
  # [
  #   -0.0583767,
  #   0.0633917,
  #   -0.0877969,
  #   0.992404,
  # ]
translation:
  [0.0, 0.0, 0.0]
  # [
  #   -0.0109111,
  #   -0.00587487,
  #   0.0243092,
  # ]
